packages/common/src/ui/layout/tables/utils/ColumnDefinitionSetBuilder.ts

```diff
@@ -65,11 +65,13 @@ const getColumnLookup = <T extends DomainObject>(): Record<
     key: 'name',
     label: 'label.name',
     width: 75,
+    sortable: false,
   },
   invoiceNumber: {
     key: 'invoiceNumber',
     label: 'label.invoice-number',
     width: 50,
+    sortable: false,
```

Comment:
also surprised to see the invoice number and total non-sortable. Are these API limitations that I didn't notice?

Reply:
Yea - I've added them to the spreadsheet, and now to our mock server